### DISCUSSION TITLE
Add option for NET_ADMIN capability to Linux node kind

### DISCUSSION THREAD

**Discussion by sflow-rt** [srl-labs/containerlab#792] [Ideas] [answered]
The tc command is a key component of Linux switchdev support for network ASICs, e.g. [DentOS](https://dent.dev/dentos/). NET_ADMIN capability is needed to use the tc command in a Linux node kind so that it can emulate a white box switch running Linux as the network operating system.

**Comment by LimeHat** [comment, marked as answer]
Hi!
Containerlab runs containers in privileged mode by default so all capabilities are exposed. Although the use case you mention - dentOS and other linux-based NOS - conceptually should have their own kind defined (for each NOS flavor) IMO.

Is there an issue you've encountered?

**Reply by sflow-rt** [reply]
I just checked and confirmed there are no issues with tc on a Linux node. Thanks.
